src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -92,6 +93,7 @@ object ProjectFortis extends App {
 
   // Main starts here
   val ssc = StreamingContext.getOrCreate(fortisSettings.progressDir, createStreamingContext)
+  implicit val connector: CassandraConnector = CassandraConnector(ssc.sparkContext)
```

Comment:
Nothing will bind to this here. How about putting this inside `CassandraEventsSink.apply` and making it `lazy` as well (to ensure it doesn't get re-serialized for every task)?

Reply:
Done in b889974.